### PR TITLE
fix(cookbook): deadlinks

### LINKED ISSUE #15: fix: Dead links on repo
From https://github.com/rust-bitcoin/www.rust-bitcoin.org/actions/runs/6454549635/job/17520321614:

FILE: cookbook/README.md

7 links checked.

FILE: cookbook/src/tx_taproot.md
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html#impl-Default → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/address/struct.Address.html#method.script_pubkey → Status: 404

59 links checked.

ERROR: 3 dead links found!
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html#impl-Default → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/address/struct.Address.html#method.script_pubkey → Status: 404

FILE: cookbook/src/tx_segwit-v0.md
[✖] mid-2023. → Status: 400 [Error: ENOENT: no such file or directory, access '/github/workspace/cookbook/src/mid-2023.'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'access',
  path: '/github/workspace/cookbook/src/mid-2023.'
}
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html#impl-Default → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/address/struct.Address.html#method.script_pubkey → Status: 404

[52](https://github.com/rust-bitcoin/www.rust-bitcoin.org/actions/runs/6454549635/job/17520321614#step:4:53) links checked.

ERROR: 4 dead links found!
[✖] mid-2023. → Status: 400
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html#impl-Default → Status: 404
[✖] https://docs.rs/bitcoin/latest/bitcoin/blockdata/address/struct.Address.html#method.script_pubkey → Status: 404

### PR DESCRIPTION
**Pull request by realeinherjar**
Closes #15

### CODE REVIEW

File: cookbook/src/tx_segwit-v0.md [hunk 1]
```diff
@@ -336,7 +336,9 @@ It expects a single argument of type `AsRef<[u8]>` which is a reference to a byt
 As the last step we print this to terminal using the [`println!`](https://doc.rust-lang.org/std/macro.println.html) macro.
 This transaction is now ready to be broadcast to the Bitcoin network.
 
+<!-- markdown-link-check-disable -->
 [^today]: mid-2023.
+<!-- markdown-link-check-enable -->
```
Comment (realeinherjar) [inline cookbook/src/tx_segwit-v0.md:341]: Check #15 for details

File: cookbook/src/tx_segwit-v0.md [hunk 2]
```diff
@@ -255,8 +255,8 @@ Inside the [`TxIn`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction
    It should be empty. That's why the `ScriptBuf::new()`.
 - `sequence` is the sequence number; it is a [`Sequence`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Sequence.html) type.
    We are using the [`ENABLE_RBF_NO_LOCKTIME`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Sequence.html#associatedconstant.ENABLE_RBF_NO_LOCKTIME) constant.
-- `witness` is the witness stack; it is a [`Witness`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html) type.
-   We are using the [`default`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html#impl-Default) method to create an empty witness that will be filled in later after signing.
+- `witness` is the witness stack; it is a [`Witness`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/witness/struct.Witness.html) type.
```
Comment (tcharding) [inline cookbook/src/tx_segwit-v0.md]: These, and all the links, should probably be to v0.30.0 not to latest?

Comment (realeinherjar) [reply cookbook/src/tx_segwit-v0.md]: Yes, I will update them. Good point

Comment (realeinherjar) [reply cookbook/src/tx_segwit-v0.md]: Done in ea10b995bc81ce09758b542393b07ca982e299f0

**Review by tcharding — APPROVED** [review]
ACK ea10b995bc81ce09758b542393b07ca982e299f0

**Comment by tcharding** [comment]
Deployed! FWIW deploying by way of the `build.sh` script really is no work, I have to drop into a shell to run the script that we use to merge PRs anyway.

**Pull request merged**